tests/console/validate_partition_table.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary:  Validate partition table via program 'blkid'.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+
+use scheduler 'get_test_suite_data';
+use filesystem_utils qw(get_partition_table_via_blkid partition_table);
+
+sub run {
+    select_console('root-console');
+    my $errors = '';
+    my $disks = get_test_suite_data()->{disks};
+
+    my $actual;
+    my $table_type_parted = get_test_suite_data()->{table_type_parted};
```

Comment:
test data is for validate against that data and compare it with the current data from the system, in general not for providing conditions (in a similar way that we would do with settings, we try to avoid that).
The problem here is that we don't have always available blkid in all the systems, in particular in s390x.
I hope that my advice here doesn't contradict what I told you previous PR (but I can see each case different unfortunately). Things that comes to my mind...

- can be installed blkid in s390x? if so, it would be straightforward, we are not force to use any particular command and the command to use is not part of functional testing, any option is ok.
- in one line we could check if the tool is installed in the system, rpm -q or similar (there are example in the repo) and if not present, fall back to the other tool.

I'm not sure why is that and if it is only an s390x thingy, it might be, but perhaps we could find this in other systems and we don't knot the reason, that is why I would not like to have and `if s390x` because we don't know it only applies to it.

In summary you can just keep the very same code that run in the two existing modules that you borrowed below, but do some query before it to know which function you need to call, please store the function in some variable that then you call it later, instead doing the logic inside the loop.

Reply:
blkid is already installed in s390x, the problem is s390x zvm is using `dasda`, and there's no output when doing 
`blkid /dev/dasda`, I already asked developer whether it is by design or not.

Reply:
afir is by design.

Reply:
ok, then knowing the reason is fine to have the condition in the code, we already have test data for it,
if `table_type eq 'dasd'` pick a tool if not the other. You can add a comment about not having uuid and not being able to use the other tool, that should be very useful.

Reply:
updated, thanks.